certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,341 @@
+                    "type"
+                  ]
+                },
+                "notification": {
+                  "type": "object",
+                  "properties": {
+                    "text": {
+                      "type": "string",
+                      "description": "text"
+                    },
+                    "detail": {
+                      "type": "string",
+                      "description": "detail"
+                    },
+                    "sharedLink": {
+                      "type": "object",
+                      "properties": {
+                        "email": {
+                          "type": "string",
+                          "description": "email",
+                          "title": ""
+                        },
+                        "subject": {
+                          "type": "string",
+                          "description": "subject"
+                        }
+                      },
+                      "description": "sharedLink",
+                      "required": [
+                        "email"
+                      ]
+                    }
+                  },
+                  "description": "notification",
+                  "required": [
+                    "sharedLink"
+                  ]
+                },
+                "document": {
+                  "type": "object",
+                  "properties": {
+                    "templateCode": {
+                      "type": "string",
+                      "description": "templateCode",
+                      "title": ""
+                    }
+                  },
+                  "description": "document",
+                  "required": [
+                    "templateCode"
+                  ]
+                },
+                "callbackMails": {
+                  "type": "string",
+                  "description": "callbackMails"
+                }
+              },
+              "default": {
+                "groupCode": "your_group_here",
+                "workflow": {
+                  "type": "WEB"
+                },
+                "notification": {
+                  "text": "TEST Firma web",
+                  "detail": "notificado v\u00eda email",
+                  "sharedLink": {
+                    "appCode": "com.viafirma.documents",
+                    "email": "your_email_here",
+                    "subject": "TEST firma remota"
+                  }
+                },
+                "document": {
+                  "templateCode": "templateSample"
+                },
+                "callbackMails": "your_email_here"
+              },
+              "required": [
+                "workflow",
+                "document",
+                "groupCode",
+                "notification"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
```

Comment:
@anfuca Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.